server/controller/assessment/removeOriginalDataPoint.ts

```diff
@@ -0,0 +1,23 @@
+import { BaseProtocol, DB } from '@server/db'
+import { OriginalDataPointRepository } from '@server/repository'
+import { Assessment, Cycle, OriginalDataPoint } from '@meta/assessment'
+
+export const removeOriginalDataPoint = async (
+  props: {
+    assessment: Assessment
+    assessmentCycle: Cycle
+    originalDataPoint: OriginalDataPoint
+  },
+  client: BaseProtocol = DB
+): Promise<OriginalDataPoint | null> => {
```

Comment:
why can this return null ? if we try to remove an odp that doesn't exist, we should get back an exception

Reply:
ok